x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_about_rule/index.tsx

```diff
@@ -280,31 +284,41 @@ const StepAboutRuleComponent: FC<StepAboutRuleProps> = ({
               }}
             />
             <EuiSpacer size="l" />
-            <CommonUseField
-              path="author"
-              componentProps={{
-                idAria: 'detectionEngineStepAboutRuleAuthor',
-                'data-test-subj': 'detectionEngineStepAboutRuleAuthor',
-                euiFieldProps: {
-                  fullWidth: true,
-                  isDisabled: isLoading,
-                  placeholder: '',
-                },
-              }}
-            />
+            <TooltipFormWrapper
```

Comment:
It could be implemented a bit simpler. EuiToolTip's `content` is optional. A tooltip isn't shown when there is no content. So instead of creating a new component author and license fields could be wrapped in `EuiToolTip` all the time where the content is defined by `ruleSource?.type === 'external'` condition.